iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergSerDe.java

```diff
@@ -138,7 +139,7 @@ public void initialize(Configuration conf, Properties serDeProperties,
     }
 
     this.projectedSchema =
-        projectedSchema(conf, serDeProperties.getProperty(Catalogs.NAME), tableSchema, jobConf);
+        projectedSchema(conf, serDeProperties.getProperty(CatalogUtils.NAME), tableSchema, jobConf);
```

Comment:
maybe we should keep this Catalogs? looks more logical `Catalogs.NAME`

Reply:
Reverted.